homeassistant/components/kaiterra/air_quality.py

```diff
@@ -0,0 +1,120 @@
+"""Support for Kaiterra Air Quality Sensors."""
+from homeassistant.components.air_quality import AirQualityEntity
+
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from homeassistant.const import CONF_DEVICE_ID, CONF_NAME
+
+from .const import (
+    DOMAIN,
+    ATTR_VOC,
+    ATTR_AQI_LEVEL,
+    ATTR_AQI_POLLUTANT,
+    DISPATCHER_KAITERRA,
+)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the air_quality kaiterra sensor."""
+    if discovery_info is None:
+        return
+
+    api = hass.data.get(DOMAIN)
+    name = discovery_info.get(CONF_NAME)
+    device_id = discovery_info.get(CONF_DEVICE_ID)
+
+    async_add_entities([KaiterraAirQuality(api, name, device_id)])
+
+
+class KaiterraAirQuality(AirQualityEntity):
+    """Implementation of a Kaittera air quality sensor."""
+
+    def __init__(self, api, name, device_id):
+        """Initialize the sensor."""
+        self._api = api
+        self._name = f"{name} - Air Quality"
```

Comment:
Space will be converted to underscore when constructing the entity_id. This is better:
```suggestion
        self._name = f"{name} Air Quality"
```